abort/index.js

```diff
@@ -0,0 +1,88 @@
+const { createEvent, scopeBind, createDomain } = require('effector');
+
+class AbortedError extends Error {
+  aborted = true;
+  key = undefined;
+  constructor(key) {
+    super(String(key));
+    this.key = key;
+  }
+}
+
+const createDefer = ({ aborter, key }) => {
+  const defer = { key };
+
+  defer.req = new Promise((rs, rj) => {
+    defer.rs = rs;
+    defer.rj = (error) => {
+      if (error instanceof AbortedError) {
+        aborter.handlers.forEach((f) => f());
+      }
+
+      rj(error);
+    };
+  });
+
+  return defer;
+};
+
+const isomorphicScopeBind = (event) => {
```

Comment:
Lol, it's so popular solution. Should we move it to `effector` package?

Reply:
If this is popular approach, I think, `effector` should export getter for `forkPage`. It will look much cleaner (and work faster, I guess), than try-catch:
```javascript
const isomorphicScopeBind = (event) => {
  if (getForkPage()) {
    return scopeBind(event)
  }
  return event
}
```

Reply:
Hmm, i'm not sure, if its a good idea, because it allows weird access to effector internals 🤔

I think, maybe, that just some additional field in `scopeBind` config would be enough, like this
```ts
const isomorphicScopeBind = (event) => scopeBind(event, { allowNoScope: true })
// just returns same event, if scope is not available
```

Reply:
Maybe it shouldn't throw error at all? :) Like, if there is no given scope, and no current `forkPage` — just return given unit unchanged?
Is there any use case for throwing error in `scopeBind`?

Reply:
I guess, there are edge cases, when scope can be missed, if scopeBind used in the wrong way 🤷

Reply:
Removed scopeBind in https://github.com/effector/patronum/pull/168/commits/98084a63a0bef20c6e34c23998ba323ba8b91246